From bc43229e85486d0bd46c36f943370dd0fd5360ab Mon Sep 17 00:00:00 2001
From: dgog <dgog@ea.com>
Date: Mon, 29 Feb 2016 16:02:43 -0800
Subject: [PATCH] Adding PSAPI flag

---
 EACallstack.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/EACallstack.build b/EACallstack.build
index 9485a96..b7c9d2c 100644
--- a/EACallstack.build
+++ b/EACallstack.build
@@ -145,6 +145,7 @@
         ${property.value}
         ${EACallstack.configDefines}
         UTF_USE_EAASSERT=1
+        PSAPI_VERSION=1                   <!--xp/vista fix -->
     </property>
 
     <property name="runtime.EACallstack.defines" if="${Dll} and ${config-compiler} == 'vc'">
-- 
1.9.4.msysgit.2

