--- //global_online/origin/packages/DirtySDK/8.17.0-origin/core/include/telemetryapi.h	2013-06-06 22:03:40.000000000 -0700
+++ //global_online/origin/packages/DirtySDK/8.17.0.1-origin/core/include/telemetryapi.h	2013-06-06 23:50:11.000000000 -0700
@@ -36,7 +36,7 @@
 #define TELEMETRY_EVENTSTRINGSIZE_DEFAULT     (16)
 
 // Extended event type
-#define TELEMETRY3_EVENTSTRINGSIZE_DEFAULT    (256)
+#define TELEMETRY3_EVENTSTRINGSIZE_DEFAULT    (4096)
 
 //! Default size for telemetry event 3 buffer
 #define TELEMETRY3_BUFFERSIZE                 (4096)
