From 3b25481b57461c18e088ab83d3e2a8072beb2b41 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jos=C3=A9=20Rivero?= <jrivero@ea.com>
Date: Wed, 27 Nov 2013 13:33:50 -0800
Subject: [PATCH] Surfacing oayload string when socket is disconnected from the
 server

---
 source/proto/protowebsocket.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/source/proto/protowebsocket.c b/source/proto/protowebsocket.c
index 076cb47..df2aa5c 100644
--- a/source/proto/protowebsocket.c
+++ b/source/proto/protowebsocket.c
@@ -1599,6 +1599,11 @@ int32_t ProtoWebSocketStatus(ProtoWebSocketRefT *pWebSocket, int32_t iSelect, vo
     // return close reason
     if (iSelect == 'crsn')
     {
+        if ((pBuffer != NULL) && (pWebSocket->iCtrlDataLen > 0))
+        {
+            int32_t iCopyLen = (iBufSize < pWebSocket->iCtrlDataLen) ? iBufSize : pWebSocket->iCtrlDataLen;
+            memcpy(pBuffer, pWebSocket->strCtrlData, iCopyLen);
+        }
         return(pWebSocket->iCloseReason);
     }
 
-- 
1.8.3.msysgit.0

